### ISSUE TITLE
getAllCourses() does not return courses for all subjects

### ISSUE THREAD

**Issue by studioph** [schedulemaker/bannerjs#1]
```
var Banner = require('banner');
var banner = new Banner(202003);
var data = await banner.getAllCourses();
```

Results in only `ACCT` and `AS` courses being returned, and duplicated many times.

**Comment by studioph** [comment]
Likely due to #2

**Comment by studioph** [comment]
Fixed in [#9](https://github.com/schedulemaker/bannerjs/pull/9)
